kolibri/plugins/management/assets/src/actions.js

```diff
@@ -13,13 +13,61 @@ function setSelectedGroupId({ dispatch }, id) {
   dispatch('SET_SELECTED_GROUP_ID', id);
 }
 
-const classroomCollection = Kolibri.resources.ClassroomResource.getCollection();
-const learnerGroupCollection = Kolibri.resources.LearnerGroupResource.getCollection();
-const learnerCollection = Kolibri.resources.FacilityUserResource.getCollection();
-const memberCollection = Kolibri.resources.MembershipResource.getCollection();
+const ClassroomResource = Kolibri.resources.ClassroomResource;
```

Comment:
While you're in here, feel free to delete any code that's not relevant for the MVP (e.g. anything to do with classrooms and groups.) I'm going to have to do that eventually anyway.